ports/hello-imgui/portfile.cmake

```
@@ -0,0 +1,164 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
+# production_mode is set to OFF during development, to test the latest version of hello_imgui
+set(production_mode ON)
+
+if(production_mode)
+    vcpkg_from_github(
+        OUT_SOURCE_PATH SOURCE_PATH
+        REPO pthom/hello_imgui
+        REF 40adf4bef4e0cbac4dfcbde1a40a79966a2ba5ec
+        SHA512 28f3293483dae0b178f8d0648d6b5c58fec979bc297d3af1c780d2714fb3fbe5ca22a314e19b3bf43d933b57f4ac26b98f9c67a4e9da22599c1d95eebe5d05b5
+        HEAD_REF master
+    )
+else()
+    set(VCPKG_USE_HEAD_VERSION ON CACHE BOOL "" FORCE)
+    vcpkg_from_git(
+        OUT_SOURCE_PATH SOURCE_PATH
+        URL https://github.com/pthom/hello_imgui
+        HEAD_REF master
+    )
+endif()
```

```suggestion
vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO pthom/hello_imgui
    REF 40adf4bef4e0cbac4dfcbde1a40a79966a2ba5ec
    SHA512 28f3293483dae0b178f8d0648d6b5c58fec979bc297d3af1c780d2714fb3fbe5ca22a314e19b3bf43d933b57f4ac26b98f9c67a4e9da22599c1d95eebe5d05b5
    HEAD_REF master
)
```
There is `vcpkg install --head <port>` to fetch the `HEAD_REF`.
IMO there shouldn't be more dev stuff in the official port.